packages/replay-next/components/Icon.tsx

```diff
@@ -173,6 +174,9 @@ export default function Icon({
       path =
         "M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z";
       break;
+    case "empty-placeholder":
```

Comment:
Hm. This feels like an odd choice to me. We can just use a spacer `<div />` with `width: 1rem; height: 1rem;` which I think is more semantically clear.

Reply:
Looks like it needs to be `1.5rem` to line up, but yeah.